Objects/setobject.c

```diff
@@ -2184,21 +2310,25 @@ set_vectorcall(PyObject *type, PyObject * const*args,
     }
 
     if (nargs) {
-        return make_new_set(_PyType_CAST(type), args[0]);
+        PyObject *rv;
+        Py_BEGIN_CRITICAL_SECTION(args[0]);
+        rv = make_new_set(_PyType_CAST(type), args[0]);
+        Py_END_CRITICAL_SECTION();
+        return rv;
     }
 
     return make_new_set(_PyType_CAST(type), NULL);
 }
 
 static PySequenceMethods set_as_sequence = {
-    set_len,                            /* sq_length */
+    (lenfunc)set_len,                            /* sq_length */
     0,                                  /* sq_concat */
     0,                                  /* sq_repeat */
     0,                                  /* sq_item */
     0,                                  /* sq_slice */
     0,                                  /* sq_ass_item */
     0,                                  /* sq_ass_slice */
-    (objobjproc)set_contains,           /* sq_contains */
+    (objobjproc)set_sq_contains,           /* sq_contains */
```

Comment:
The comments should be aligned with each other. Or, alternatively, this could be written to use C99 designated initializers:

```
static PySequenceMethods set_as_sequence = {
    .sq_length = (lenfunc)set_len,
    .sq_contains = (objobjproc)set_sq_contains.
};
```

Reply:
Just align the comments and keep the PR focused on its core task and minimize the diff as much as possible.  Changing variable names, rewriting logic, or making stylistic changes to long stable code is just a distractor and will make the edit harder to review.